src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,63 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    private final int[] result = new int[ReportColumn.values().length];
+
     public void getStatistic(String fromFileName, String toFileName) {
+        calculation(read(fromFileName));
+        File fileWrite = new File(toFileName);
+        clearFile(toFileName);
+        write(fileWrite);
+    }
+
+    private void clearFile(String path) {
+        try (FileWriter clearFile = new FileWriter(path)) {
+            clearFile.write("");
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private void calculation(List<String> list) {
+        for (String line : list) {
+            String[] operation = line.split(",");
```

Comment:
this array contains the operation name and amount, so why don't you use this amount instead of line.substring() method call ?